.NET/Microsoft.Recognizers.Text.Number/Parsers/BaseNumberParser.cs

```diff
@@ -22,6 +22,9 @@ public BaseNumberParser(INumberParserConfiguration config)
         {
             this.Config = config;
 
+            //As soon as you develop on a machine that does not use en-US as standard-culture, numbertests for the other languages fail

```

Comment:
I understand the final correction for this bug is not in the repo yet and this causes issues, but please don't push the code with this workaround.
Sorry for the delay to have a final fix integrated.